### ISSUE TITLE
Consider tagging a new release

### ISSUE THREAD

**Issue by matrizzo** [pali/igmpproxy#88]
#81 has been merged and igmpproxy can now run as an unprivileged user. It would be great if you could tag a new release so that distros pick up this change.

Thanks!

**Comment by AfromD** [comment]
I am also waiting on a newer release that includes a commit from July 2021. Would be nice to have an updated release.

**Comment by pali** [comment]
Done! Now I have released new version 0.4 from master branch.

**Comment by AfromD** [comment]
Thanks so much!
